v3.1/glfw/native_darwin.go

```diff
@@ -1,22 +1,29 @@
 package glfw
 
-//#define GLFW_EXPOSE_NATIVE_COCOA
-//#define GLFW_EXPOSE_NATIVE_NSGL
-//#include "glfw/include/GLFW/glfw3.h"
-//#include "glfw/include/GLFW/glfw3native.h"
-import "C"
+/*
+#define GLFW_EXPOSE_NATIVE_COCOA
+#define GLFW_EXPOSE_NATIVE_NSGL
+#include "glfw/include/GLFW/glfw3.h"
+#include "glfw/include/GLFW/glfw3native.h"
```

Comment:
Off-topic, but why is GitHub showing so much has changed in this diff when in fact it hasn't:

![image](https://cloud.githubusercontent.com/assets/1924134/9560306/b471596c-4dc1-11e5-9f4b-7e30000683c3.png)